packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -18,10 +18,6 @@ const handlePropValidation = ({href, tabIndex, children}: AnchorProps): void =>
     );
   }
 
-  if (children == null) {
```

Comment:
For the time being, let's leave this error in.  Some consumers of this component might be in a pure JS ecosystem, so they won't get the typescript error.  I'm considering doing a TS -> React.propTypes thing at some point.

Reply:
I added that back in.